tox.ini

```diff
@@ -20,16 +22,24 @@ deps =
     pytest34: pytest>=3.4,<3.5
     pytest37: pytest>=3.7,<3.8
     pytest39: pytest>=3.9,<3.10
+    pytest310: pytest>=3.10,<3.11
     termcolor>=1.1.0
     supported-xdist: pytest-xdist>=1.14
     unsupported-xdist: pytest-xdist<1.14
-    pytest-rerunfailures
+    rerunfailures: pytest-rerunfailures
```

Comment:
If we create this rule, should we activate it somewhere too?

Reply:
Oh here: https://github.com/Frozenball/pytest-sugar/pull/160/files#diff-354f30a63fb0907d4ad57269548329e3R24 I see :+1: